packages/lwc/scripts/utils/rollup.js

```diff
@@ -58,7 +46,6 @@ function rollupConfig(config) {
                         'process.env.NODE_ENV': JSON.stringify('production'),
                         preventAssignment: true,
                     }),
-                rollupFeaturesPlugin(prod),
```

Comment:
In the `lwc` package, this plugin was useless. It wasn't actually changing any files.

In the future, if we want to use feature flags in something like `@lwc/wire-service`, we should run the plugin there rather than in `lwc`. In `lwc` the JS files are huge, so it's pretty expensive to run the Babel plugin on those files anyway.